.gitignore

```diff
@@ -2,7 +2,7 @@
 
 *.exe
 src/bitcoin
-src/bitcoind
+src/bitcoinabc
```

Comment:
Not trying to be silly, but this directory should probably be named src/bitcoinabcd to indicate that it's the daemon directory.  I would suggest changing src/bitcoin to src/bitcoinabc as well, but I'm not sure if that makes for an annoying merging experience *if* any improvements from bitcoin core show up.  Thoughts?

Reply:
Don't the issues of two nodes running on the same computer come from location of ~/.bitcoin and from running on port 8333